pkg/webhooks/rules.go

```diff
@@ -1,21 +1,21 @@
 package webhooks
 
-import "k8s.io/api/admissionregistration/v1beta1"
+import v1 "k8s.io/api/admissionregistration/v1"
```

Comment:
Just a question, what is the reason why in the previous file we call the import by 
```admissionv1 "k8s.io/api/admissionregistration/v1"```
and in this file with an import name of `v1` ?

Reply:
It's just an alias. Probably in the file where it was aliased to `admissionv1` it was done to avoid collision with other packages also named `v1`